java/src/jmri/jmrit/display/layoutEditor/LayoutTurnout.java

```diff
@@ -1542,11 +1542,9 @@ public void setCoordsCenter(Point2D p) {
     }
 
     private void reCalculateCenter() {
-        double centreX = (pointC.getX() - pointA.getX()) / 2;
-        double centreY = (pointC.getY() - pointA.getY()) / 2;
-        centreX = pointA.getX() + centreX;
```

Comment:
@rhwood Aha, there's a change in sign in the expression - that slipped past me! But my comment was more due to the PR only mentioning the new combobox, rather than also an update to this method.